packages/package_info_plus/package_info_plus_windows/pubspec.yaml

```diff
@@ -13,7 +13,7 @@ dependencies:
   ffi: ^2.0.1
   flutter:
     sdk: flutter
-  win32: ^2.7.0
+  win32: ^3.0.0
```

Comment:
Please don't set artificial range restrictions on widely-used packages. See [this comment](https://github.com/ThexXTURBOXx/flutter_web_auth_2/commit/3a7b846677f1fd4326a7a641607a1ba8941dca18#r82923160) and [this comment](https://github.com/flutter/plugins/pull/6352#discussion_r961905860)

Reply:
Done